apps/api/src/modules/providers/sms-plivo/sms-plivo.service.ts

```diff
@@ -10,7 +10,25 @@ export interface SmsPlivoData {
 export interface SmsPlivoResponseData {
   api_id: string;
   message: string;
-  message_uuid: string[];
+  messageUuid: string;
+}
+
+export interface PlivoMessageStatusResponse {
+  api_id: string;
+  error_code: string;
+  from_number: string;
+  message_direction: string;
+  messageState: string;
+  message_time: string;
+  message_type: string;
+  message_uuid: string;
+  resource_uri: string;
+  to_number: string;
+  total_amount: string;
+  total_rate: string;
+  units: number;
+  status: string;
+  // add any other fields you expect in the response
```

Comment:
Do we expect any other fields here?

Reply:
Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/261/commits/5ce4e67bd9877772054c24dac5a90f5ae9c00aad based on test run and response obtained